src/c/admin-project-details-card.js

```diff
@@ -0,0 +1,91 @@
+window.c.AdminProjectDetailsCard = (function(m, h){
+  return {
+    controller: function(args) {
+      var project = args.resource,
+          generateStatusText = function() {
+            var statusTextObj = m.prop({}),
+                statusText = {
+                  online: {cssClass: 'text-success', text: 'NO AR'},
+                  successful: {cssClass: 'text-success', text: 'FINANCIADO'},
+                  failed: {cssClass: 'text-error', text: 'NÃO FINANCIADO'},
+                  waiting_funds: {cssClass: 'text-waiting', text: 'AGUARDANDO'},
+                  rejected: {cssClass: 'text-error', text: 'RECUSADO'},
+                  draft: {cssClass: '', text: 'RASCUNHO'},
+                  in_analysis: {cssClass: '', text: 'EM ANÁLISE'},
+                  approved: {cssClass: 'text-success', text: 'APROVADO'}
+                };
+
+            statusTextObj(statusText[project.state]);
+
+            return statusTextObj;
+          },
+          generateRemaingTime = function() {
+            var remainingTextObj = m.prop({}),
+                translatedTime = {
+                  days: 'dias',
+                  minutes: 'minutos',
+                  hours: 'horas',
+                  seconds: 'segundos'
+                };
+
+            remainingTextObj({
+              unit: translatedTime[project.remaining_time.unit || 'seconds'],
+              total: project.remaining_time.total
+            });
+
+            return remainingTextObj;
+          };
+
+      return {
+        project: project,
+        statusTextObj: generateStatusText(),
+        remainingTextObj: generateRemaingTime()
+      };
+    },
+
+    view: function(ctrl) {
+      var project = ctrl.project,
+          progress = project.progress.toFixed(2),
+          statusTextObj = ctrl.statusTextObj(),
+          remainingTextObj = ctrl.remainingTextObj();
+
+      return m('.project-details-card.card.u-radius.card-terciary.u-marginbottom-20', [
+        m('div', [
+          m('.fontsize-small.fontweight-semibold', [
+            m('span.fontcolor-secondary', 'Status:'),' ',m('span', {class: statusTextObj.cssClass}, statusTextObj.text),' '
+          ]),
+          (function(){
```

Comment:
I think using a ternary here would make this view a bit cleaner, while maintaining consistency with other components. 

```
project.is_published ? component.stuff : ''
```